packages/roosterjs-editor-core/lib/test/editor/EditorTest.ts

```diff
@@ -92,6 +92,19 @@ describe('Editor insertContent()', () => {
         expect(editor.getContent()).toBe('<div id="text">hellotext</div>');
     });
 
+    it('insert begin with multiple nodes', () => {
+        // Act
+        editor.insertContent('hello<br>world', {
+            position: ContentPosition.Begin,
+            updateCursor: true,
+            replaceSelection: true,
+            insertOnNewLine: false,
+        });
+
+        // Assert
+        expect(editor.getContent()).toBe('<div id="text">hello<br>worldtext</div>');

```

Comment:
is that `text` supposed to be there? The test passes with it?

Reply:
Yes. See the test case above, it also has a "text"